README.md

```diff
@@ -2,6 +2,16 @@
   <img width="400" alt="proof" src="./packages/docs/src/media/proof.color.text.svg"/>
 </h1>
 
+<p align="center">
+  <a href="#"><img alt="npms" src="https://img.shields.io/npm/v/npm"></a>
```

Comment:
Thanks for taking on this work. Looking at the contents of the badges, I don't think this reflects the current version of the `@proof-ui` packages on npm. Do you know what this is pulling from?

Reply:
Sorry about this, I just updated it with the current version of @proof-ui, which is v0.1.5.